tests/MongoDBDriver_APM-001.phpt

```diff
@@ -0,0 +1,578 @@
+--TEST--
+HHVM-190: APM Specification
+--FILE--
+<?php
+include dirname(__FILE__) . '/utils.inc';
+
+$m = new MongoDB\Driver\Manager("mongodb://localhost:27017");
+cleanup( $m );
+
+class MySubscriber implements MongoDB\Driver\Monitoring\CommandSubscriber
+{
+	public function commandStarted( \MongoDB\Driver\Monitoring\CommandStartedEvent $event )
+	{
+		echo "started:\n";
+		var_dump( $event );
+	}
+
+	public function commandSucceeded( \MongoDB\Driver\Monitoring\CommandSucceededEvent $event )
+	{
+		echo "succeeded:\n";
+		var_dump( $event );
+	}
+
+	public function commandFailed( \MongoDB\Driver\Monitoring\CommandFailedEvent $event )
+	{
+		echo "failed:\n";
+		var_dump( $event );
+	}
+}
+
+$m->addSubscriber( new MySubscriber() );
+cleanup( $m );
+
+$d = new \MongoDB\BSON\Decimal128("1234.5678");
+
+$bw = new MongoDB\Driver\BulkWrite( [ 'ordered' => false ] );
+$_id = $bw->insert( [ 'decimal' => $d ] );
+$r = $m->executeBulkWrite( 'demo.test', $bw );
+
+$query = new MongoDB\Driver\Query( [] );
+$cursor = $m->executeQuery( "demo.test", $query );
+var_dump( $cursor->toArray() );
+?>
+--EXPECTF--
+started:
+object(MongoDB\Driver\Monitoring\CommandStartedEvent)#8 (6) {
```

Comment:
Yup. I missed that. Although this doesn't come from user land, so it *ought* to be safe.